doc/user_guide/tools.rst

```diff
@@ -453,6 +453,7 @@ features differ from numpy):
 
```

Comment:
I suggest to add a section to this chapter of the User Guide explaining how AxesManager manages the units and the new ``convert_units`` ``AxesManager`` and ``DataAxis`` methods.